bin/garden-build

```diff
@@ -13,25 +13,28 @@ export REPO_ROOT="$(readlink -f "${thisDir}/..")"
 
 commitid="${commitid:-local}"
 eval "$dgetopt"
+
+# Update all param flags
 while true; do
 	flag="$1"; shift
 	dgetopt-case "$flag"
 	case "$flag" in
-		--debug) debug=1 ;;	# for jumping in the prepared image"
-		--ports) ports=1 ;;	# for using "debian-ports"
-		--arch) arch="$1"; shift ;; # for adding "--arch" to garden-init
-		--qemu) qemu=1 ;;	# for using "qemu-debootstrap"
-		--features) features="$1"; shift ;; # adding features
-		--disable-features) disablefeatures="$1"; shift ;; # ignoring features
-		--suite) suite="$1"; shift ;; # suite is a parameter this time
-		--gardenversion|--timestamp) version="$1"; shift ;; # timestamp is a parameter this time
-		--suffix) suffix="$1"; shift ;; # target name prefix
-		--prefix) prefix="$1"; shift ;; # target name suffix
-		--commitid) commitid="$1"; shift ;; # build commit hash
-		--commitid-long) commitid_long="$1"; shift ;; # build commit hash, untruncated - used for /etc/os-release only
-		--skip-tests) tests=0 shift ;; # skip tests
-		--userid) userID="$1" shift ;;
-		--usergid) userGID="$1" shift ;;
+		--debug)			debug=1 ;;			# for jumping in the prepared image"
+		--debian-mirror)		debianMirror=$1 ; shift ;; 	# for adding Debian native mirrors (only dev; default: 0)
```

Comment:
Input value `$1` is not used. Debian repo is hardcoded to `echo "deb https://deb.debian.org/debian testing main"` in file `bin/garden-debian-sources-list`
```suggestion
		--debian-mirror)		debianMirror=1 ;; 	# for adding Debian native mirrors (only dev; default: 0)
```

since this is a developer parameter, i would say to keep the parameter simple and use a hard coded value